pymc/distributions/timeseries.py

```diff
@@ -201,7 +253,25 @@ class GaussianRandomWalk(distribution.Continuous):
         distribution for initial value (Defaults to Flat())
     """
 
+    rv_op = gaussianrandomwalk
+
+    @classmethod
+    def dist(
+        cls,
+        mu: Optional[Union[np.ndarray, float]] = None,
```

Comment:
Fair point, I copied them from TruncatedNormal. Are these ones wrong too? If so I can update them too

https://github.com/pymc-devs/pymc/blob/main/pymc/distributions/continuous.py#L601

Reply:
Also int are also valid. Because there's so many we should probably save the union of these type hints in a variable that we can reuse a across distributions

Reply:
> Got it, I copied them from TruncatedNormal. Are these ones wrong too? If so I can update them too
> 
> https://github.com/pymc-devs/pymc/blob/main/pymc/distributions/continuous.py#L601

Yes they are also incomplete

Reply:
Ok ill open up another PR just for this to catch up the distributions on main :D